package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@skyux/popovers",
-  "version": "4.1.1",
+  "version": "4.0.0",
```

Comment:
Downgrade?

Reply:
Yikes, good catch. That must have come from a bad merge.